### ISSUE TITLE
Docker full installation doesn't work

### ISSUE THREAD

**Issue by Omicron166** [smicallef/spiderfoot#1643]
I built the container with the command `docker-compose -f docker-compose.yml -f docker-compose-full.yml up` and it throws this:

```
Creating spiderfoot ... done
Attaching to spiderfoot
spiderfoot    | /usr/bin/env: ‘python3\r’: No such file or directory
```

**Comment by smicallef** [comment]
I can't reproduce this. Are you definitely using the latest version? Could you share the contents of both files and the version of docker you're using?

**Comment by Omicron166** [comment]
Docker version 20.10.14, build a224086
docker-compose version 1.29.2, build 5becea4c
Docker Desktop 4.7.0 (77141)
All over WSL 2 on windows 10.0.19044.1620

The wsl backend works and i already ran virtual machines on this pc.
Also, the image from `docker build -t spiderfoot .` works fine, so this isn't the problem

**Comment by Omicron166** [comment]
Update: `docker-compose -f docker-compose.yml -f docker-compose-dev.yml up` also throws the same error
only `docker-compose -f docker-compose.yml up` works

**Comment by smicallef** [comment]
Are you using the latest version of SpiderFoot from the master branch? Please also ensure you kill any running `spiderfoot` container, build, and then run the `docker-compose`:
1. `docker ps` -> kill any `spiderfoot` containers
2. `cd` to your `spiderfoot` installation (cloned repo) folder.
3. `docker build -t spiderfoot .`
4. `docker-compose -f docker-compose.yml -f docker-compose-dev.yml up`

Following the above steps on a Linux machine works successfully for me.

**Comment by Omicron166** [comment]
I keep track of the containers thought docker desktop GUI, that isn't the problem.

The container build is successfull, it fails when running spiderfoot.
I can even restart the docker-compose container.

**Comment by Omicron166** [comment]
I just updated docker and refreshed the repo, the bug still being in the master branch

Creating spiderfoot ... done
Attaching to spiderfoot
spiderfoot    | /usr/bin/env: 'python3\r': No such file or directory

**Comment by smicallef** [comment]
I'm wondering if this issue is specific to Windows. Have you been able to try this on a non-Windows system?

**Comment by Omicron166** [comment]
Nope, i only tested under windows, i'll try to build it on a linux virtual machine

**Comment by RufatRoot** [comment]
The same for me on windows machine. Don't have ability to test on Linux.
